.travis.yml

```diff
@@ -6,8 +6,21 @@ cache:
   directories:
    - $HOME/.m2
 
-# skip install
-install: true
+install:
```

Comment:
I'm not convinced caching will improve it much because it is actually a lot of data. I'm going to leave this for a future build speed optimization task.